### ISSUE TITLE
use VS code as git text editor can‘t rebase -i

### ISSUE THREAD

**Issue by dengdairiluo** [Microsoft/vscode#66146]
Issue Type: <b>Bug</b>

 When I use  git rebase -i HEAD~5 VS code show nothing. when I change to vim git shows what I need.

VS Code version: Code 1.30.1 (dea8705087adb1b5e5ae1d9123278e178656186a, 2018-12-18T18:05:00.654Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2700)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.36GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (25)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.23.0
xml|Dot|2.4.0
gitlens|eam|9.3.0
vscode-great-icons|emm|2.1.46
json-tools|eri|1.0.2
nbpreviewer|jit|1.2.2
vscode-fileheader|mik|0.0.2
vscode-language-pack-zh-hans|MS-|1.30.2
anaconda-extension-pack|ms-|1.0.1
python|ms-|2018.12.1
cpptools|ms-|0.20.1
csharp|ms-|1.17.1
team|ms-|1.144.1
vscode-gitignore-generator|pio|1.0.1
quicktype|qui|12.0.46
java|red|0.36.0
vscode-yaml|red|0.2.0
vscode-icons|rob|8.0.0
markdown-preview-enhanced|shd|0.3.11
vscode-java-debug|vsc|0.16.0
vscode-java-dependency|vsc|0.3.0
vscode-java-pack|vsc|0.5.0
vscode-java-test|vsc|0.13.0
vscode-maven|vsc|0.13.0
vim|vsc|1.0.1

</details>

**Comment by bpasero** [comment]
@dengdairiluo does `git config -e --global` open it in VSCode?

**Comment by dengdairiluo** [comment]
Yes. I use "git config --global core.editor code" to set vscode. when I use "git rebase -I"
<img width="630" alt="1" src="https://user-images.githubusercontent.com/10458959/50838273-978bf780-1398-11e9-8f4b-38d9ef05b505.png">
<img width="883" alt="2" src="https://user-images.githubusercontent.com/10458959/50838276-9c50ab80-1398-11e9-8f86-d4c9b149971e.png">

**Comment by dengdairiluo** [comment]
When I use vim,
<img width="646" alt="image" src="https://user-images.githubusercontent.com/10458959/50838371-d457ee80-1398-11e9-83de-e1a5d5e2b2f0.png">

**Comment by bpasero** [comment]
@dengdairiluo so you are saying that the file opens in VSCode in both cases but simply in the one case the editor is empty?
